tools/cell_census_builder/anndata.py

```diff
@@ -114,7 +114,7 @@ def _filter(ad: anndata.AnnData, retain_X: Optional[bool] = True) -> anndata.Ann
         if organism_ontology_term_id is not None:
             obs_mask = obs_mask & (ad.obs.organism_ontology_term_id == organism_ontology_term_id)
         if assay_ontology_term_ids is not None:
-            obs_mask = obs_mask & ad.obs.assay_ontology_term_id.isin(RNA_SEQ)
+            obs_mask = obs_mask & ad.obs.assay_ontology_term_id.isin(assay_ontology_term_ids)
```

Comment:
This is a bug in the current version of the code, as per discussion with @bkmartinjr yesterday.